commitlint.config.js

```diff
@@ -7,6 +7,7 @@ const scopes = [
   'badge',
   'build',
   'button',
+  'button-group',
```

Comment:
`angular:button-group`

Reply:
The scope itself is fine. In order the commit message to get to `fix(angular:button-group)` this line should be changed to `angular:button-group`.

Based on the new commit strategy most of the time the commit has to have a `project` and a `scope` so the release tool can get them and set them in the generated changelog. Otherwise a manual change will be required on the changelog generated list

Reply:
Right, but my point is 1. This is on the angular branch and 2) none of the other entries have this. 
Since this isn't part of the bugfix. I'll revert.